src/test/java/org/takes/tk/TkProxyTest.java

```diff
@@ -50,30 +51,43 @@
 public final class TkProxyTest {
 
     /**
-     * TkProxy can work.
+     * An array of http methods for testing.
+     */
+    private static final String[] METHODS = {
```

Comment:
@dmzaytsev The requirement was not clear for me :) 
I added the methods except( CONNECT and HEAD, they need a different testing).
Should I add the testcase for them too?